src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -32,8 +32,8 @@ azure-mgmt-compute==27.1.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==9.1.0
 azure-mgmt-containerregistry==10.0.0
-azure-mgmt-containerservice==20.2.0
-azure-mgmt-core==1.3.1
+azure-mgmt-containerservice==20.3.0
+azure-mgmt-core==1.3.2
```

Comment:
> msrest==0.7.1
> azure-mgmt-core==1.3.2

Since this PR includes the upgrade of widely used core packages, we recommend that this PR be merged at the beginning of the next sprint to avoid unexpected risks to CLI. So I label this PR with `do not merge` tag first